renovate.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": [
-    "config:base"
+    "config:base",
+    ":maintainLockFilesWeekly",
```

Comment:
What is the default behaviour without this option?

Reply:
No lockfile maintenance PR.

Reply:
Fair enough.